WordPress/Classes/ViewRelated/Stats/Extensions/Double+Stats.swift

```diff
@@ -46,6 +42,13 @@ extension Double {
         }
     }
 
+    func formatWithCommas() -> String {
```

Comment:
I'm seeing what appears to be a bit of a hot spot, performance-wise. This method appears to be called from more & more places, and with it, there are numerous instantiations with each invocation:

1. `NumberFormatter`
1. `NSNumber`
1. `Double`, when invoked from the `Int` extension.

Are you seeing the same thing? If so, do you think we might want to limit the formatter instantiations? If not, perhaps it's something we keep an eye on?